CONTRIBUTING.md

```diff
@@ -318,10 +337,17 @@ These tests require several dependencies that can be installed either from the R
 | unidiff        | python3-unidiff (Zesty and higher) | unidiff        |
 | yamllint       | yamllint                          | yamllint       |
 
+#### Quick testing
+
 There is a tool [scripts/check_rosdep](./scripts/check_rosdep.py) which will check most formatting errors such as alphabetization and correct formatting.
-It is recommended to run it before submitting your contribution.
+It will not check for all errors, but it is a quick way to check for common errors.
+
+Here is a one liner to check all files in the [rosdep](./rosdep) directory:
+```bash
+for f in ./rosdep/*.yaml; do echo "$f"; ./scripts/check_rosdep.py "$f"; done
+```
 
-For example, to check a change to `rosdep/base.yaml`:
+Or simply use `pytest` (although the output is quite verbose):
 ```bash
-python3 scripts/check_rosdep.py rosdep/base.yaml
+pytest test/rosdep_formatting_test.py
 ```
```

Comment:
Honestly, I'm not sure we need this bit at all.  This is already covered by the general testing above, so maybe we can just drop this stanza completely?

Reply:
Yeah, this was basically a leftover that I didn't know if I should've deleted.
I'm fine for dropping.